challenges.md

```diff
@@ -1,68 +1,110 @@
 # About the Code Challenges
-We have a couple of code challenges designed to give you the opportunity to showcase your skills. We love to see creativity, originality and simplicity of design. All of our projects involve some creativity, research, interest in data, automation and of course amazing instructions. Please make sure to write the most incredible and informative README.md files you can imagine.
+Welcome to the Contrast Code Challenges project page. Code challenges are an important part of our recruiting and hiring process. Every engineer/developer, designer or technologist that works at Contrast, full-time or intern submits a code challenge as part of the process to join the team.
 
-There are tons of amazing open and free data sets to play with. Here are a few for inspiration:
+The code challenge is designed to give candidates the ability to showcase their skills in a low stress, extended time model rather than in a conference room as part of a technical interview. While we give you ample time, each code challenge is designed to take no more than 4 to 8 hours. We believe that technologists work best in the comfort of their own home, coffee shop or favorite place to code or design.
 
-* [Data.gov](https://www.data.gov/open-gov/)
-* [NYC Open Data](https://nycopendata.socrata.com/)
-* [Austin, TX Data](https://data.austintexas.gov/browse)
-* [AWS Open Data](https://aws.amazon.com/public-data-sets/)
-* [OpenData Baltimore](https://data.baltimorecity.gov/)
-* [Open Data Sets on GitHub](http://www.kdnuggets.com/2015/04/awesome-public-datasets-github.html)
-* [Firebase Data Sets](https://www.firebase.com/docs/open-data/)
+Most importantly we do the challenge for you. Anyone submitting a code challenge, owns their own code, whether they put it on GitHub or another site. Our theory is that you are putting time into the interview process. We want you to get something out of the effort. What better than something to add to your portfolio?
```

Comment:
Comma after "importantly". "Do" the challenge? How about "create" the challenge? No comma after "Anyone submitting a code challenge"